modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryServiceImpl.java

```diff
@@ -408,7 +408,7 @@ protected void activate(
 		_portletResourcePermissionsServiceTrackerMap =
 			ServiceTrackerMapFactory.openSingleValueMap(
 				bundleContext, PortletResourcePermission.class,
-				"(&(com.liferay.object=true)(resource.name=*))",
+				"(&(com.liferay.object.definition=true)(resource.name=*))",
```

Comment:
Could you please check if we can use here the `ObjectConstants.RESOURCE_NAME_OBJECT_DEFINITION`?